releasenotes/notes/fixspanevents-35586b155aada1c3.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    APM: fix issue where new span field for span events wasn't properly serialized when received as message pack.
```

Comment:
```suggestion
    APM: fix issue where a new span field for span events wasn't properly serialized when received as a message pack.
```

Reply:
Thanks! I updated the release note a bit from this suggestion as I realized when looking it up that message pack is written as [MessagePack](https://msgpack.org/) (as it's the name of the format)